app/common/lib/menuUtil.js

```diff
@@ -88,9 +88,13 @@ const createBookmarkTemplateItems = (bookmarks, parentFolderId) => {
         label: site.get('customTitle') || site.get('title') || site.get('location'),
         click: (item, focusedWindow, e) => {
           if (eventUtil.isForSecondaryAction(e)) {
-            CommonMenu.sendToFocusedWindow(focusedWindow, [messages.SHORTCUT_NEW_FRAME, site.get('location'), { openInForeground: !!e.shiftKey }])
+            appActions.createTabRequested({
+              url: site.get('location'),
+              windowId: focusedWindow.id,
+              active: !!e.shiftKey
+            })
           } else {
-            CommonMenu.sendToFocusedWindow(focusedWindow, [messages.SHORTCUT_ACTIVE_FRAME_LOAD_URL, site.get('location')])
+            appActions.loadURLInActiveTabRequested(focusedWindow.id, site.get('location'))
```

Comment:
with a small change to `windowState` we can use -2 here as well. Maybe add windowState.WINDOW_ID_CURRENT to match chrome.windows api?